docs/install_guides/ncs2-setup-guide.md

```diff
@@ -20,3 +22,25 @@ Once you have your Intel® Distribution of OpenVINO™ toolkit installed, follow
 
 You've completed all required configuration steps to perform inference on Intel® Neural Compute Stick 2. 
 Proceed to the <a href="#get-started">Start Using the Toolkit</a> section to learn the basic OpenVINO™ toolkit workflow and run code samples and demo applications.
+
+@sphinxdirective
+
+.. _ncs guide macos:
+
+@endsphinxdirective
+
+## macOS
+
+These steps are required only if you want to perform inference on Intel® Neural Compute Stick 2
+powered by the Intel® Movidius™ Myriad™ X VPU. For more details, see also the
+[Get Started page for Intel® Neural Compute Stick 2](https://software.intel.com/en-us/neural-compute-stick/get-started).
```

Comment:
Can we rid of this outdated page?

Reply:
let me synch with @mlyashko regarding this